README.md

```diff
@@ -1 +1,134 @@
-# hello-react-front-end
\ No newline at end of file
+<div align="center">
+<img src= "Logo.png">
+</div>
+
+
+<div align="center">
+  <h3><b>Hello_React_Front-end </b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Hello-React-Front-end ](#-rails-react-hello-app-)
+  - [Link To Backend](#link-to-Backend)  
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 Hello-Rails-Front-end <a name="about"></a>
+
+The **Hello-Rails-Front-end**  is an application just a sample hello react API.
+## Link to Backend  <a name="built-with"></a>
+- Link to Backend: [Hello-React-Back-end](https://github.com/AliShahSafdari/hello-rails-back-end)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- JAVASCRIPT
+- CSS
+- REACT
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **Setup React**
+- [ ] **Add API**
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- git
+- Ruby
+- Rails
+- Bundle gem
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:AliShahSafdari/hello-react-front-end.git
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails s
+```
+
+### Run tests
+
+To run tests, run the following commands:
+
+```sh
+  rubocop
+  npm run stylelint
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **AliShahSafdari**
+
+- GitHub: [AliShahSafdari](https://github.com/AliShahSafdari)
+- Linkedin: [AliShahSafdari](https://www.linkedin.com/in/AliShahSafdari)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Stats dashboard**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/AliShahSafdari/hello-rails-front-end/issues).
```

Comment:
- [ ] Your issues page current shows a 404 error page please fix it 👍🏿

Reply:
I fixed the link problem.